src/Parallel/Tags/ResourceInfo.hpp

```diff
@@ -0,0 +1,65 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <string>
+
+#include "Options/Options.hpp"
+#include "Parallel/ResourceInfo.hpp"
+#include "Utilities/PrettyType.hpp"
+#include "Utilities/System/ParallelInfo.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace Parallel {
+namespace OptionTags {
+/// \ingroup ParallelGroup
+/// Options group for resource allocation
+template <typename Metavariables>
+struct ResourceInfo {
+  using type = Parallel::ResourceInfo<Metavariables>;
+  static constexpr Options::String help = {"Options for allocating resources."};
+};
+}  // namespace OptionTags
+
+namespace Tags {
+/// \ingroup DataBoxTagsGroup
```

Comment:
I personally think the docs is cleaner not sticking everything into the groups, so my suggestion is to not put these tags into the groups. We definitely haven't been consistent, but as the number of tags has increased, putting them all in the Doxygen group I've found to make the doxygen group very clutter :shrug:

Reply:
I removed them from the DataBoxTags group, but left them in the Parallel group because there are less things in the Parallel group and because I think they are important enough tags that they should be discoverable with the rest of the parallel things.